web/src/main/ui/app/components/flows-new/edit-flow/ui/stepper.component.ts

```diff
@@ -59,9 +59,14 @@ export class StepperComponent extends CdkStepper implements OnChanges, AfterCont
     }
   }
   setStatus(status: string) {
-    let runStatus = status.replace('_', ' ');
-    runStatus = runStatus.replace('-', ' ');
-    this.status = runStatus.split(' ');
+    if (status) {
+      let runStatus = status.replace('_', ' ');
+      runStatus = runStatus.replace('-', ' ');
+      this.status = runStatus.split(' ');
+    } else {
+      this.status = '';
```

Comment:
Maybe it makes better sense to have empty array [] instead of ''. This will be more consistent.

Reply:
Good catch @alexsmr